src/screens/Loading/index.js

```diff
@@ -1,31 +1,31 @@
 import * as React from 'react';
 import {ActivityIndicator, Modal, StyleSheet, View} from 'react-native';
-import { withInteractionsManaged } from '../../components/WithInteractionManaged';
+import {withInteractionsManaged} from '../../components/WithInteractionManaged';
 import {COLORS} from '../../utils/theme';
 
 const Loading = ({visible}) => {
   return (
     <Modal transparent visible={visible}>
       <View style={styles.container}>
         <View style={styles.content}>
-          <ActivityIndicator size="large" color={COLORS.holyTosca} />
+          <ActivityIndicator size="large" color={COLORS.holytosca} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constant `COLORS.holyTosca` has been renamed to `COLORS.holytosca`. This change could potentially introduce bugs if the `COLORS.holytosca` constant does not exist in the `COLORS` object. Please ensure that the correct constant is being used.
```diff
- <ActivityIndicator size="large" color={COLORS.holyTosca} />
+ <ActivityIndicator size="large" color={COLORS.holytosca} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->